Frontend/src/4_features/EditableArticleCard/ui/EditableArticleCardHeader/EditableArticleCardHeader.tsx

```
@@ -76,8 +76,8 @@ export const EditableArticleCardHeader = memo(
           result = await dispatch(saveArticleData()).unwrap();
         }
         console.log(result);
-        if (result && result?.article?.id) {
-          navigate(getRouteArticleDetails(result.article.id));
+        if (result && result?.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)